src/sidebar/config/build-settings.js

```diff
@@ -4,6 +4,8 @@ import * as postMessageJsonRpc from '../util/postmessage-json-rpc';
 
 /**
  * @typedef {import('../../types/config').ConfigFromHost} ConfigFromHost
+ * @typedef {import('../../types/config').AnnotatorConfigFromHost} AnnotatorConfigFromHost
+ * @typedef {import('../../types/config').EmbedderConfigFromHost} EmbedderConfigFromHost
```

Comment:
Perhaps we should use the terms `ConfigFromAnnotator` and `ConfigFromEmbedder` to refer to these refinements of `ConfigFromHost`. This would be a little more succinct and also make it clear that the use of Annotator / Embedder is primarily about where the configuration comes from rather than _which_ configuration it contains.

Reply:
I prefer your naming. I'm going to make this change.